common/src/main/java/org/opensearch/ml/common/transport/connector/MLConnectorDeleteRequest.java

```diff
@@ -22,24 +24,41 @@
 import lombok.Builder;
 import lombok.Getter;
 
+@Getter
 public class MLConnectorDeleteRequest extends ActionRequest {
-    @Getter
-    String connectorId;
+    private final String connectorId;
+    private final String tenantId;
```

Comment:
making this `final` creates a lot of extra noise elsewhere.

Reply:
Could you please clarify in more detail what kind of noise you are referring to?

Reply:
Having to do the else block below, for example.